### PR TITLE
Bump cdi to version v1.9.0

### PR DESCRIPTION
**Pull request by annastopel**
/bugfix #70
Moving CDI vendor code and image to v1.9.0

### CODE REVIEW

File: tools/manifest-templator/manifest-templator.go [hunk 1]
```diff
@@ -260,7 +260,7 @@ func getCDI(data *templateData) {
 	// Get CDI Deployment
 	cdideployment := cdicomponents.GetDeployment(
 		"kubevirt",
-		"v1.7.0",
+		"v1.9.0",
```
Comment (djzager) [inline tools/manifest-templator/manifest-templator.go]: Should also run `./hack/build-manifests.sh` and include those changes in this PR.

Comment (annastopel) [reply tools/manifest-templator/manifest-templator.go]: @djzager 
done:
now pr contains  genarated manifests:
```
deploy/converged/olm-catalog/kubevirt-hyperconverged/0.0.1/kubevirt-hyperconverged-operator.v0.0.1.clusterserviceversion.yaml
deploy/converged/operator.yaml

```

**Comment by djzager** [comment]
@tiraboschi are you able to have a look at this PR since @rthallisey is out?

**Pull request merged**